NitroxModel-Subnautica/DataStructures/GameLogic/Buildings/Rotation/Metadata/BaseModuleRotationMetadata.cs

```diff
@@ -23,13 +23,13 @@ protected BaseModuleRotationMetadata() : base(typeof(BaseAddModuleGhost))
 
         public BaseModuleRotationMetadata(Int3 cell, int direction) : base(typeof(BaseAddModuleGhost))
         {
-            Cell = cell.Model();
+            Cell = cell.ToDto();
             Direction = direction;
         }
 
         public override string ToString()
         {
-            return "[BaseModuleRotationMetadata cell: " + Cell + " direction: " + Direction + " ]";
+            return $"[BaseModuleRotationMetadata - Cell: {Cell}, Direction: {Direction}]";
```

Comment:
I would suggest that this change is made inline with AnchoredFaceRotationMetadata
return $"[AnchoredFaceRotationMetadata Cell: {Cell} Direction: {Direction} FaceType: {FaceType}]";